posthog/api/test/test_organization.py

```diff
@@ -103,6 +103,41 @@ def test_cannot_update_organization_if_not_owner_or_admin(self):
         self.organization.refresh_from_db()
         self.assertNotEqual(self.organization.name, "ASDFG")
 
+    def test_plugin_alerts_list(self):
+        self.organization_membership.level = OrganizationMembership.Level.OWNER
+        self.organization_membership.save()
+        user2 = self._create_user("user2@posthog.com")
+        user3 = self._create_user("user3@posthog.com")
+
+        response = self.client.patch(
+            f"/api/organizations/{self.organization.id}",
+            {"members_to_send_plugin_alerts": [self.user.uuid, user2.uuid]},
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.json()["members_to_send_plugin_alerts"], [str(self.user.uuid), str(user2.uuid)])
+        self.organization_membership.refresh_from_db()
+        self.assertEqual(self.organization_membership.send_plugin_alerts, True)
```

Comment:
non-blocking

Rather than checking the db directly, we can GET the organization and this was we are checking the API interface which e.g. the frontend is interacting with, and freeing ourselves to change the implementation details, i.e. it will be easier to make changes